pkg/cnidel/cnidel.go

```diff
@@ -164,6 +164,11 @@ func getExecCniParams(cniType string, netInfo *danmtypes.DanmNet, ep *danmtypes.
     "CNI_ARGS="        + os.Getenv("CNI_ARGS"),
     "CNI_PATH="        + os.Getenv("CNI_PATH"),
   }
+  if cniType == "weave-net" {
```

Comment:
yeah the general aim would be not oversharing if something is not required
but if now it looks like there are some CNIs depending on PATH, I'm okay with unconditionally passing it over to all CNIs
in fact I would prefer that over hard-coding CNI-type specific conditions into the code. it can happen other, so far untested CNIs also depend on the same parameter

Reply:
Sure, seems PATH is a relatively common environment variable (althoug I only found it needed by weave), so I will make it unconditionally passed to all CNI plugins.